### DISCUSSION TITLE
Retrospect v5.7.17

### DISCUSSION THREAD

**Discussion by basrieter** [retrospect-addon/plugin.video.retrospect#1833] [Announcements]
Fixed some minor channel issues. 

### Framework related
_None_

### GUI/Settings/Language related
_None_

### Channel related
* Fixed: Older url's would have the incorrect build-version of the UR Play app (Fixes #1827).
* Added: NPO profiles and related stuff (Fixes #1828).
* Fixed: NPO Search.

<hr /><em>This discussion was created from the release <a href='https://github.com/retrospect-addon/plugin.video.retrospect/releases/tag/v5.7.17'>Retrospect v5.7.17</a>.</em>

**Comment by jcmulder** [comment]
When opening a NPO live stream the app first gives the message "Login Failed, Could not logon to the selected channel" , after it the stream plays correct.
Why I get a login failed-message? I have a clean install and I have also tried to fill in my NPO account settings , but the message remains. 
Anyone has an advise?
 I am not sure if this is the right place for this message, then I hope someone can help me finding the right place.

**Reply by basrieter** [reply]
Well, if you set up a propper username/password, it should just work! Keep in mind that Retrospect stores your password safely in an encrypted vault secure by a PIN. I see a lot of users mistake the PIN for the site password. 

When setting a password, the Vault PIN is asked first, then the password that needs to be stored. Whenever Retrospect needs to use the password it will only ask the user for the PIN and it will use that PIN to decrypt the password and then use it.

I see someone reported something similar: https://github.com/retrospect-addon/plugin.video.retrospect/issues/1835

**Reply by jcmulder** [reply]
Thanks a lot. I will look again.

**Reply by jcmulder** [reply]
Bas, het is gelukt. Gek dat het mis ging. Dank. Groet, Jeroen.

**Reply by basrieter** [reply]
Graag gedaan! Veel plezier met Retrospect.
